### PR TITLE
Adds a confirmation prompt to all OOC events in the secrets panel.

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do
Adds a confirmation prompt to all OOC events in the secrets panel.

## Why It's Good For The Game
One misclick shouldn't cause an entire round to be done for.

## Images of changes
image

## Testing
Go to the secrets panel
Try to turn all sprites into revolvers
Get a confirmation prompt

## Changelog
:cl:
tweak: Added a confirmation prompt to all OOC events in the secrets panel.
/:cl:

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -2801,7 +2801,7 @@
 
 	else if(href_list["secretsfun"])
 		if(!check_rights(R_SERVER|R_EVENT))	return
-
+		if(alert("Are you sure you want to run this secret?","Confirmation","Yes","Cancel") != "Yes") return
```
Comment (Henri215) [inline code/modules/admin/topic.dm:2804]:
```suggestion
		if(alert("Are you sure you want to run this secret?", "Confirmation", "Yes", "Cancel") != "Yes")
			return
```

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -2801,7 +2801,8 @@
 
 	else if(href_list["secretsfun"])
 		if(!check_rights(R_SERVER|R_EVENT))	return
```
Comment (DGamerL) [inline code/modules/admin/topic.dm:2803]: I know you didn't touch this, but would you mind making this compliant with our standards?

**Review by DGamerL — CHANGES_REQUESTED** [review]
Just this small thing, and see warriorstar's review!

**Review by S34NW — COMMENTED** [review]
Outstanding changes with no commits in 2 weeks, please resolve

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -2801,7 +2801,8 @@
...
-
+		if(alert("Are you sure you want to run this secret?", "Confirmation", "Yes", "Cancel") != "Yes")
```
[the remark this answers is not in the document]
Comment (Adrer) [reply code/modules/admin/topic.dm:2804]: Has been addressed.

File: code/modules/admin/topic.dm [hunk 4]
```diff
@@ -3059,20 +3089,32 @@
...
 			if("onlyme")
+				var/sure = alert(usr, "Are you sure you want to do this?", "Confirmation", "Yes", "No")
+				if(sure == "No")
+					return
 				SSblackbox.record_feedback("tally", "admin_secrets_fun_used", 1, "Only Me")
 				usr.client.only_me()
 			if("onlyoneteam")
+				var/sure = alert(usr, "Are you sure you want to do this?", "Confirmation", "Yes", "No")
+				if(sure == "No")
+					return
```
Comment (Contrabang) [inline code/modules/admin/topic.dm:3117]: Can you make these all into one-liners and turn them into `!= "Yes"` so if they close it without clicking no, it wont go through?
```suggestion
				if(alert(usr, "Are you sure you want to do this?", "Confirmation", "Yes", "No") != "Yes"
					return
```

**Review by DGamerL — APPROVED** [review]
Contra's review addressed, LGTM

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**